tests/baselines/reference/docker/azure-sdk.log

```diff
@@ -0,0 +1,95 @@
+[@azure/service-bus] started
+Warning: You have changed the public API signature for this project. Updating review/service-bus.api.md
+[@azure/storage-blob] started
+5 of 18: [@azure/storage-blob] completed successfully in ? seconds
+[@azure/storage-datalake] started
+6 of 18: [@azure/storage-datalake] completed successfully in ? seconds
+[@azure/storage-file] started
+7 of 18: [@azure/storage-file] completed successfully in ? seconds
+[@azure/storage-queue] started
+8 of 18: [@azure/storage-queue] completed successfully in ? seconds
+[@azure/template] started
+9 of 18: [@azure/template] completed successfully in ? seconds
+[@azure/core-http] started
+10 of 18: [@azure/core-http] completed successfully in ? seconds
+[@azure/core-paging] started
+11 of 18: [@azure/core-paging] completed successfully in ? seconds
+[@azure/event-processor-host] started
+12 of 18: [@azure/event-processor-host] completed successfully in ? seconds
+[testhub] started
+13 of 18: [testhub] completed successfully in ? seconds
+[@azure/identity] started
+14 of 18: [@azure/identity] completed successfully in ? seconds
+[@azure/core-amqp] started
+[@azure/keyvault-certificates] started
+15 of 18: [@azure/keyvault-certificates] completed successfully in ? seconds
+[@azure/keyvault-keys] started
+16 of 18: [@azure/keyvault-keys] completed successfully in ? seconds
+[@azure/keyvault-secrets] started
+17 of 18: [@azure/keyvault-secrets] completed successfully in ? seconds
+
+SUCCESS (16)
+================================
+@azure/abort-controller (? seconds)
+@azure/core-http (? seconds)
+@azure/core-paging (? seconds)
+@azure/cosmos (? seconds)
+@azure/event-hubs (? seconds)
+@azure/event-processor-host (? seconds)
+@azure/identity (? seconds)
+@azure/keyvault-certificates (? seconds)
+@azure/keyvault-keys (? seconds)
+@azure/keyvault-secrets (? seconds)
+@azure/storage-blob (? seconds)
+@azure/storage-datalake (? seconds)
+@azure/storage-file (? seconds)
+@azure/storage-queue (? seconds)
+@azure/template (? seconds)
+testhub (? seconds)
+================================
+
+SUCCESS WITH WARNINGS (1)
+================================
+@azure/service-bus (? seconds)
+Warning: You have changed the public API signature for this project. Updating review/service-bus.api.md
+================================
+
+FAILURE (1)
+================================
+@azure/core-amqp (? seconds)
+>>> @azure/core-amqp
+tsc -p . && rollup -c 2>&1
+src/errors.ts(579,20): error TS7053: Element implicitly has an 'any' type because expression of type 'any' can't be used to index type 'typeof ConditionErrorNameMapper'.
+src/errors.ts(600,34): error TS7053: Element implicitly has an 'any' type because expression of type 'any' can't be used to index type 'typeof SystemErrorConditionMapper'.
+src/errors.ts(601,20): error TS7053: Element implicitly has an 'any' type because expression of type 'any' can't be used to index type 'typeof ConditionErrorNameMapper'.
```

Comment:
any idea why this happens?

Reply:
Because they're legitimately indexing an `enum` with `any`, apparently - `ConditionErrorNameMapper[condition as any]`. Error seems legit to me.